### ISSUE TITLE
Missing Dependencies?

### ISSUE THREAD

**Issue by KazooieTheBird** [Ethanol10/Ror2Ganon#5]
Tried to clone this and build it but am just seeing this in visual studio:
![image](https://github.com/Ethanol10/Ror2Ganon/assets/41878986/5550b088-2182-4eb0-88eb-4df87d3df399)

Had no issues doing this with a different player character mod. Did you forget to upload something to this repo?

**Comment by Ethanol10** [comment]
No, you need to link the assemblies in the project, currently those dependencies are not linked so they can't be found. Youll need to find these dependencies and add them in yourself. In the manifest file inside the thunderstorms upload, there should be a list of dependencies that you need to include( the version usually doesn't matter)

**Comment by Ethanol10** [comment]
[BepInDependency("com.bepis.r2api", BepInDependency.DependencyFlags.HardDependency)]
    [BepInDependency("com.bepis.r2api.prefab", BepInDependency.DependencyFlags.HardDependency)]
    [BepInDependency("com.bepis.r2api.language", BepInDependency.DependencyFlags.HardDependency)]
    [BepInDependency("com.bepis.r2api.sound", BepInDependency.DependencyFlags.HardDependency)]
    [BepInDependency("com.bepis.r2api.networking", BepInDependency.DependencyFlags.HardDependency)]
    [BepInDependency("com.weliveinasociety.CustomEmotesAPI", BepInDependency.DependencyFlags.SoftDependency)]
    [BepInDependency("com.DestroyedClone.AncientScepter", BepInDependency.DependencyFlags.SoftDependency)]
    [BepInDependency("com.ThinkInvisible.ClassicItems", BepInDependency.DependencyFlags.SoftDependency)]

    [BepInDependency("com.rune580.riskofoptions", BepInDependency.DependencyFlags.SoftDependency)]


Both hard and soft dependencies are required to build the package correctly.

**Comment by Ethanol10** [comment]
You can retrieve these packages on thunderstore, and linking the plugins to your project.
